service_user.go

```diff
@@ -11,11 +11,19 @@ import (
 type (
 	// ServiceUser is the representation of a Service User in the Aiven API.
 	ServiceUser struct {
-		Username   string `json:"username"`
-		Password   string `json:"password"`
-		Type       string `json:"type"`
-		AccessCert string `json:"access_cert"`
-		AccessKey  string `json:"access_key"`
+		Username                    string         `json:"username"`
+		Password                    string         `json:"password"`
+		Type                        string         `json:"type"`
+		AccessCert                  string         `json:"access_cert"`
+		AccessKey                   string         `json:"access_key"`
+		AccessCertNotValidAfterTime string         `json:"access_cert_not_valid_after_time"`
+		ServiceControl              ServiceControl `json:"service_control"`
```

Comment:
access_control, not service_control

Reply:
Also this needs to be omitted except when it's being set for Redis.

Reply:
Ups sorry did change the name of the field